### PR TITLE
perf(uploads): Set parallelism to 1 for zones multiput uploads

### PR DESCRIPTION
**Pull request by klk-**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @klk- has signed the CLA. Thanks for the pull request!

File: src/api/uploads/MultiputUpload.js [hunk 1]
```diff
@@ -203,6 +203,11 @@ class MultiputUpload extends BaseMultiput {
         const uploadUrl = this.getBaseUploadUrlFromPreflightResponse(preflightResponse);
         let createSessionUrl = `${uploadUrl}/files/upload_sessions`;
 
+        // Parallelism is currently detrimental to multiput upload performance in Zones, so set it to 1.
+        if (createSessionUrl.indexOf('fupload-ec2') >= 0) {
```
Comment (priyajeet) [inline src/api/uploads/MultiputUpload.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Comment (klk-) [reply src/api/uploads/MultiputUpload.js]: Done

**Comment by klk-** [comment]
@jstoffan I can't tell if the tests themselves or failing or the test runner is.  When I click on "Details" the page I get says "We couldn't find the repository box/box-ui-elements".  Do you know what's going on?

**Comment by jstoffan** [comment]
@klk-, the [logs](https://travis-ci.org/box/box-ui-elements/jobs/469731944) seem correct to me. The most recent commit message doesn't conform to the linting requirements.

**Pull request merged**